lldb/test/API/functionalities/breakpoint/breakpoint_options/TestBreakpointOptions.py

```diff
@@ -90,6 +90,15 @@ def breakpoint_options_language_test(self):
             num_expected_locations=1,
         )
```

Comment:
run_break_set_by_symbol returns the break ID of the new breakpoint.  You could look it up, get its location, get the function from the symbol and get its mangled name.  That way you could do a mangling scheme independent test that you produced the right mangled name.

Reply:
Nice! this works really well